lms/static/certificates/sass/_views.scss

```diff
@@ -11,3 +11,35 @@
 // #DISTINCTION CERT
 // ------------------------------
 
+
+// ------------------------------
+// #INVALID
+// ------------------------------
+
+.wrapper-content.status-invalid {
+    @extend %layout-wrapper;
+
+    .wrapper-content-grid {
+        @extend %layout;
+    }
+
+  .content-main {
+    margin-bottom: spacing-vertical(base);
+
+    @include susy-breakpoint($bp-medium, $susy) {
+      @include span(9 of 12 first);
+    }
+  }
+
+  .content-secondary {
+    margin-top: spacing-vertical(x-small);
```

Comment:
NIT: If you can stick to setting margins in one direction (per [CSS Wizardry's philosophy](http://csswizardry.com/2012/06/single-direction-margin-declarations/) which we have agreed with in [our standards](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#marginvisual-space)), I'd prefer it. You have bottom defined elsewhere here and in general, FYI.